src/components/Demo.svelte

```diff
@@ -69,65 +70,148 @@ let languages: {
   },
 ] as const;
 
-const urlParams = new URLSearchParams(window.location.search);
-if (urlParams.has("go")) {
-  languageCode.Go = LZString.decompressFromEncodedURIComponent(
-    urlParams.get("go"),
-  );
-}
-if (urlParams.has("c")) {
-  languageCode.C = LZString.decompressFromEncodedURIComponent(
-    urlParams.get("c"),
-  );
-}
-if (urlParams.has("c++")) {
-  languageCode["C++"] = LZString.decompressFromEncodedURIComponent(
-    urlParams.get("c++"),
-  );
-}
-if (urlParams.has("python")) {
-  languageCode.Python = LZString.decompressFromEncodedURIComponent(
-    urlParams.get("python"),
-  );
-}
-if (urlParams.has("typescript")) {
-  languageCode.TypeScript = LZString.decompressFromEncodedURIComponent(
-    urlParams.get("typescript"),
-  );
-}
-if (urlParams.has("tsx")) {
-  languageCode.TSX = LZString.decompressFromEncodedURIComponent(
-    urlParams.get("tsx"),
-  );
-}
+type ShareParameters = {
+  version: number;
+  language: string;
+  code: string;
+  fontSize: string;
+  simplify: boolean;
+  flatSwitch: boolean;
+  highlight: boolean;
+  colors: ColorList;
+};
+
+// ADD-LANGUAGES-HERE
+const languageAliases: Record<string, Language> = {
+  go: "Go",
+  c: "C",
+  cpp: "C++",
+  python: "Python",
+  typescript: "TypeScript",
+  tsx: "TSX",
+};
+
+const reverseLanguageAliases = Object.fromEntries(
+  Object.entries(languageAliases).map(([alias, lang]) => [lang, alias]),
+);
 
+const currentVersion: number = 1; // Needs to be updated when a new version is released.
+let fontSize = $state("1em");
 let simplify = $state(true);
 let flatSwitch = $state(true);
 let highlight = $state(true);
+let parsedColorList = undefined;
+const urlParams = new URLSearchParams(window.location.search);
+let isLanguageInURL: boolean = false;
+let selection = $state(languages[0]);
+
+if (urlParams.has("language")) {
+  try {
+    isLanguageInURL = true;
+    const urlLanguage = urlParams.get("language");
+    const languageAlias = languageAliases[urlLanguage];
+    const languageMatched = languages.find(
```

Comment:
Grammar nitpick - `matchedLanguage` not `languageMatched`.

But, since we care more about the user's intent than the implementing code - maybe `requestedLanguage`?